### DISCUSSION TITLE
FYI: walrus is another alternative

### DISCUSSION THREAD

**Discussion by zhangcheng** [brainix/pottery#3] [General]
http://charlesleifer.com/blog/walrus-lightweight-python-utilities-for-working-with-redis/

**Comment by brainix** [comment]
Thanks for the heads up, @zhangcheng!  Walrus looks amazing, but I think that it's a different project from Pottery.  It looks like Walrus is more ambitious, while Pottery tries to stay closer to the Python container APIs.

I hadn't heard about Walrus before, and I appreciate the tip!
